test/raxx/stack_test.exs

```diff
@@ -43,8 +43,12 @@ defmodule Raxx.StackTest do
     end
   end
 
+  defmodule DefaultMiddleware do
+    use Middleware
+  end
+
   test "a couple of NoOp Middlewares don't modify the response of a simple controller" do
-    middlewares = [{NoOp, :irrelevant}, {NoOp, 42}]
+    middlewares = [{NoOp, :irrelevant}, {NoOp, 42}, {DefaultMiddleware, :nothing}]
```

Comment:
I don't think this is enough of a test. I'd test like 3 cases in middleware_test.exs for this.

Reply:
I'm not sure exactly which three cases you wanted here, but I am now using `use` for more than one test and am reasonably happy it's covered.
Tests for the router stuff will add even more cases